content/index.html

```diff
@@ -51,15 +51,15 @@
   <div class="outerbox">
     <div class="box">
 
-      <p><a href="https://giantswarm.io/"><img src="/giantswarm_logo_standard_white.svg" width="100" height="50" /></a></p>
+      <p><a href="https://giantswarm.io/"><img src="giantswarm_logo_standard_white.svg" width="100" height="50" /></a></p>
 
       <h1>Hello World</h1>
 
       <p>Congratulations! You have just deployed and started a little something on Giant Swarm.</p>
 
       <p>Now go ahead and learn how easy it is to create and run your own service <a href="https://docs.giantswarm.io/guides/your-first-service/">in your favorite language</a>.</p>
 
-      <p>Or share your pride and let the world know by <a target="_blank" href="https://twitter.com/share?text=I've just deployed HelloWorld on @giantswarm!"/>tweeting this</a>!</p>
+      <p>Or share your pride and let the world know by <a target="_blank" href="https://twitter.com/share?url=https://giantswarm.io/&amp;text=I've just deployed my first service on @giantswarm!"/>tweeting this</a>!</p>
```

Comment:
maybe instead of `on @giantswarm` we could do `on my own @giantswarm #Kubernetes cluster` or sth like that